### DISCUSSION TITLE
Release on every build?

### DISCUSSION THREAD

**Discussion by andrew-welker** [PepperDash/Essentials#940] [CI/CD]
After trying to find a certain Essentials release, I'm rethinking creating a Github Release for every build we do using the CI/CD pipeline. Its becoming noise to some extent. I'm thinking beta builds should still create a release (builds from the development branch). For using certain Essentials builds in plugins, Nuget would be the preferred delivery mechanism. For downloading an .cpz to install on a processor, the build.zip artifact can be downloaded from a workflow run. Commits would still be tagged with the release number in this scenario. Thoughts?

**Comment by ngenovese11** [comment]
Just my 2 cents; I still prefer to have everything as a package in NUGET

**Reply by andrew-welker** [reply]
Right, we wouldn't stop doing that. We'd stop creating a new Github Release for every build though.

**Reply by ngenovese11** [reply]
Then I agree!
